### ISSUE TITLE
Tabbed Files Header -- could show one or two more files?

### ISSUE THREAD

**Issue by zeffii** [enjalot/blockbuilder#118]
![wuzka](https://cloud.githubusercontent.com/assets/619340/12752730/2420ccb0-c9c4-11e5-87ff-0b5ad78dd35d.png)

In this case (and i'm sure many in the future) the `index.html` has 
```javascript
<body>
  <svg></svg>
  <script src="dillitant.js"></script>
</body>
```
and the `js` does a d3.csv search for `sugars.csv`. but both are tucked away in the dropdown, while really they are the only two files i'm interested in for a while. Meaning that switching between the files is a mouse-jockey afair? unless there's some kind of tab-roulette key to rotate through files.?

is there any way to relegate the readme.MD and thumbnail to the dropdown instead? or increase the amount of files shown in the header/tabs ?

**Comment by mostaphaRoudsari** [comment]
:+1:

**Comment by enjalot** [comment]
I agree this would be great. The file tabs were hastily thrown together and I'd love help making them cleaner.

One important point is that I do want the README.md to always be there, as it's a core part of what makes a bl.ock a bl.ock, and people should be encouraged to document even just a little bit. 

As for the thumbnail.png, that should only show up if there is a thumbnail probably, and the option to add a thumbnail to your gist could be its own UI that makes itself obvious when you are on your own block and havent added a thumbnail yet.

**Comment by zeffii** [comment]
OK. reading https://github.com/enjalot/blockbuilder/blob/master/public/js/components/files.js

**Comment by enjalot** [comment]
:+1: this seems linked to #102 
Notice that the code you are reading also includes some other File related components.
The files are ultimately driven by the gist stored in the state. When we mess with the files we fire off actions which save the changes into the gist. That same gist is what gets pushed to github.

ping me on gitter with any questions :bow:
